hare/interfaces.go

```diff
@@ -15,7 +15,9 @@ type layerPatrol interface {
 
 // Rolacle is the roles oracle provider.
 type Rolacle interface {
-	Validate(context.Context, types.LayerID, uint32, int, types.NodeID, []byte, uint16) (bool, error)
+	Validate(ctx context.Context, layerID types.LayerID,
```

Comment:
why make it different from the rest of the interface?

Reply:
I will updated the whole interface to have named parameters.